lisp/mastodon-toot.el

```diff
@@ -69,10 +69,19 @@ Set `mastodon-toot--content-warning' to nil."
   (interactive)
   (mastodon-toot--kill))
 
+(defun mastodon-toot--remove-docs ()
+  "Get the body of a toot from the current compose buffer."
+  (let ((re "^|=+=|$"))
```

Comment:
I'm trying to match the literal `|` character and in emacs regexp that's just an unescaped `|` afaict. the `\|` is the regexp "or" character afaict:

given: `|=+=|`
`|` -> 2 matches
`\|` -> 5 matches (but no highlights, undefined behvavior i guess?)
`=\|+` -> 3 matches